go.work

```diff
@@ -1,4 +1,6 @@
-go 1.21
+go 1.21.4
```

Comment:
Needed to upgrade to this version because of the block-sdk see:

https://github.com/skip-mev/block-sdk/blob/v1.4.0/go.mod#L3-L5